.github/workflows/canary-integration-test.yml

```diff
@@ -576,9 +576,11 @@ jobs:
         yq merge --inplace --arrays append tests/manifests/test-cluster-on-pvc-encrypted.yaml tests/manifests/test-kms-vault-spec.yaml
         yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].count" 2
         yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].volumeClaimTemplates[0].spec.resources.requests.storage" 6Gi
+        yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.cephVersion.image" quay.io/ceph/daemon-base:latest-pacific-devel
         kubectl create -f tests/manifests/test-cluster-on-pvc-encrypted.yaml
         yq merge --inplace --arrays append tests/manifests/test-object.yaml tests/manifests/test-kms-vault-spec.yaml
         sed -i 's/ver1/ver2/g' tests/manifests/test-object.yaml
+        sed -i 's/true/false/g' tests/manifests/test-object.yaml
```

Comment:
I am changing `VAULT_SKIP_VERIFY` into `false`, not `preservePoolsOnDelete`. Is okay to remove it? When I removed it failed for OSD creation AFAIR.  So I made this change, so it won;t effect the existing case

Reply:
No it's just that the `sed` is not really clear since you are changing all the occurrences of `true` with `false`. Can you just isolate that so we know what the CI does?